tcms/testruns/tests/test_permissions.py

```diff
@@ -128,3 +128,31 @@ def verify_post_with_permission(self):
             self.assertEqual(case.history.latest().history_id, case_run.case_text_version)
             self.assertEqual(last_run.build, case_run.build)
             self.assertEqual(None, case_run.close_date)
+
+
+class TestViewExecutionCommentsTestCase(PermissionsTestCase):
```

Comment:
Name here doesn't follow the pre-existing convention but the old name is inappropriate b/c it doesn't tell you what is being tested. This doesn't have anything to do with test executions. It is the Test Run page, bulk-edit menu, "Add comment" menu item.